styles.css

```diff
@@ -0,0 +1,427 @@
+* {
+  margin: 0;
```

Comment:
- [ ] Remember, the changes stated above are applied on the mobile version too, like colors, font, and others, in addition to them, consider updating the mobile design alignment on the texts to get the same adjustments or positions as shown in this design on the left.

<img width="694" alt="8" src="https://user-images.githubusercontent.com/77843050/211077343-9be6ab2a-011f-43d7-8be1-1f4e7ba8e3ca.png">

Reply:
## Hello 👋  @joskalenda 
Thanks so much for your review. It was really thorough. You must have spent a lot of time on it. You noticed details that I didn't even consider. I was about to argue about the footer background colors before realizing that the desktop version and mobile version have different footer colors. Thanks again.

Is there a way to choose a particular reviewer? I would always choose you 😄